fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/SearchQueryRenderer.java

```diff
@@ -131,17 +131,20 @@
     // Enable use of legacy whole-system search parameters for the search request
     private final boolean legacyWholeSystemSearchParamsEnabled;
 
+    // Include DATA in the data fetch queries
+    private final boolean includeResourceData;
     /**
      * Public constructor
      * @param identityCache
      * @param rowOffset
      * @param rowsPerPage
      */
     public SearchQueryRenderer(JDBCIdentityCache identityCache,
-        int rowOffset, int rowsPerPage) {
+        int rowOffset, int rowsPerPage, boolean includeResourceData) {
```

Comment:
not new, but worth cleaning up if we're making more commits:
```suggestion
            int rowOffset, int rowsPerPage, boolean includeResourceData) {
```

Reply:
done